### DISCUSSION TITLE
Documentation on how to setup Documentation

### DISCUSSION THREAD

**Discussion by Blankf** [almenscorner/intunecd-monitor#28] [Q&A]
Hi!

Thank you for this awesome project.
do you by any chance have some documentation on how to setup the documentation properly?
i've done it locally by using IntuneCD-startdocumentation but how can we add that to the tool. 

Ferry

**Comment by almenscorner** [comment]
Hi,

Thank you ☺️

If you are not in a rush to get the documentation setup in Monitor, I would suggest holding off for just a bit. I am adding a new option to make it SUPER easy to start using the documentation in the front end as well -> https://www.linkedin.com/posts/tobiasalmen_intunecd-msintune-mvpbuzz-activity-7113501997399470081-8qyk?utm_source=share&utm_medium=member_desktop

**Reply by Blankf** [reply]
I`m not, thank you! will hold off for now :-)

Take Care, Good luck 👍

**Reply by almenscorner** [reply]
With the new features in #31 you can now generate documentation by just checking a box on your baseline tenant. Check the release notes to upgrade.
